<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenueDialog</class>
 <widget class="QDialog" name="MenueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButtonMenu">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/resources/images/icons/menue.png</normaloff>:/resources/images/icons/menue.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonSettings">
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonUpdateMangaList">
     <property name="text">
      <string>Update Manga Lists</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonClearDownloads">
     <property name="text">
      <string>Clear Downloads</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonExit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
